samples/clients.py

```diff
@@ -0,0 +1,40 @@
+from createsend import *
+
+auth = {
+  'access_token': 'YOUR_ACCESS_TOKEN',
+  'refresh_token': 'YOUR_REFRESH_TOKEN' }
+clientId = 'YOUR_CLIENT_ID'
+
+cs = CreateSend(auth)
+client = Client(auth, clientId)
+
+# Get list of sent campaigns
+print("List of sent campaigns:")

```

Comment:
Do we also want to include some samples on how to pass in the proper date format `YYYY-MM-DD` and tags too?

Reply:
I was thinking the basic enough is sufficient to display the pagination nature. Having said that, saw the ruby one and make it on par with it to have similar consistency.